lib/cache/senders/impressions_sender.rb

```diff
@@ -18,20 +18,30 @@ def call
           if ENV['SPLITCLIENT_ENV'] == 'test'
             post_impressions
           else
-            Thread.new do
-              @config.logger.info('Starting impressions service')
+            impressions_thread
 
-              loop do
-                post_impressions
-
-                sleep(randomize_interval(@config.impressions_refresh_rate))
+            if defined?(PhusionPassenger)
```

Comment:
@maksimf should this code live in this repo?

Part of me think that either:
- We put this snippet in another repo for users of passenger, or
- Provide these lines in the docs for users to add them to their set up.

Would that work?